azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,327 @@
+# Azure Cognitive Services Computer Vision SDK for Python
+
+The Computer Vision service provides developers with access to advanced algorithms for processing images and returning information. Computer Vision algorithms analyze the content of an image in different ways, depending on the visual features you're interested in. For example, Computer Vision can determine if an image contains adult or racy content, find all the faces in an image, get handwritten or printed text. This service works with popular image formats, such as JPEG and PNG. 
+
+You can use Computer Vision in your application to:
+
+- Analyze images for insight
+- Extract text from images
+- Generate thumbnails
+
+Looking for source code or API reference?
```

Comment:
For this section, how about:
___
Looking for more documentation?

* [SDK reference documentation](https://docs.microsoft.com/python/api/azure-cognitiveservices-vision-computervision/azure.cognitiveservices.vision.computervision)
* [Cognitive Services Computer Vision documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/)
___
Source:

```Markdown
Looking for more documentation?

* [SDK reference documentation](https://docs.microsoft.com/python/api/azure-cognitiveservices-vision-computervision/azure.cognitiveservices.vision.computervision)
* [Cognitive Services Computer Vision documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/)
```

Reply:
@mmacy Is there a reason you don't want the link to the pip package in this section as well?

Reply:
Added the reference link into the install section instead

Reply:
I moved the link down to the pip install section. It will be interested to see if any people in the study know/care about the pip page.